### ISSUE TITLE
error from vecmem when building tip of main

### ISSUE THREAD

**Issue by cgleggett** [acts-project/traccc#47]
When trying to build tip of main (dfcd7e7e91e1e8e4fb12d27824f9ad489c8f140b), I'm getting this error:

```
In file included from /home/leggett/work/traccc/src/core/include/definitions/algebra.hpp:14,
                 from /home/leggett/work/traccc/src/core/include/edm/cell.hpp:11,
                 from /home/leggett/work/traccc/src/core/include/algorithms/detail/sparse_ccl.hpp:10,
                 from /home/leggett/work/traccc/src/core/include/algorithms/component_connection.hpp:10,
                 from /home/leggett/work/traccc/src/examples/cpu/seq_single_module.cpp:10:
/home/leggett/work/traccc/src/core/include/definitions/primitives.hpp:13:10: fatal error: vecmem/containers/static_array.hpp: No such file or directory
 #include <vecmem/containers/static_array.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

the version of vecmem that it's pulling in is 2e06a5167e0927e6e6ed6a4f1abd5b5f552f059f

**Comment by cgleggett** [comment]
ahh - `cmake/traccc-vecmem.cmake` calls for GIT_TAG `2e06a5167e0927e6e6ed6a4f1abd5b5f552f059f`. 

This should be updated. It builds fine if you use the tip of vecmem (currently `1ccfd3a65e90b67339792ebd95aa87072bd41b64`) but I don't know if this is desirable.

**Comment by paulgessinger** [comment]
Maybe it would make sense to use a submodule for vecmem as well for now?

**Comment by krasznaa** [comment]
A submodule would be one way of doing that. Though I don't think that would make it either simpler or more complicated to change the version of VecMem than the system that we have now...

**Comment by stephenswat** [comment]
This should be solved with the merging of #48.
